src/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
+
```

Comment:
- [ ] As I mentioned in the previous comments, kindly make sure you fix all your linters errors instead of disablling them.

Reply:
@tresorsawasawa Thanks for the review. I know how important eslint is, but I disabled it because the linter errors only appeared on GitHub and not on my vscode. I didn't know how else to fix it on GitHub since I had already fixed it in my local repo.(That is, my app ran locally without errors of any sort)

Reply:
@tresorsawasawa Thanks for the review. I know how important eslint is, but I disabled it because the linter errors only appeared on GitHub and not on my vscode. I didn't know how else to fix it on GitHub since I had already fixed it in my local repo.(That is, my app ran locally without errors of any sort)